managed/models/agent_model.go

```diff
@@ -73,6 +73,8 @@ const (
 	VMAgentType                         AgentType = "vmagent"
 )
 
+var v2_42 = version.MustParse("2.41.99")
```

Comment:
I'm seeing different patterns here and [there](https://github.com/percona/pmm/blob/6aedc981c283edc91b677501a2b20890fa3bf0af/managed/services/agents/mongodb.go#L41-L43).

I wonder if we should stick to one standard, i.e. `var v2_42 = version.MustParse("2.42.0-0")`, which seems to be easier to read?

Reply:
Updated as you and Nurlan suggested.